src/nikhilMJun20/ReverseInteger.java

```diff
@@ -0,0 +1,25 @@
+/*WAP to reverse number (Consider Input as Integer not as String).
+Example:-
+Input:- 12345
+Output:- 54321
+*/
+
+package nikhilMJun20;
+
+public class ReverseInteger {
+	void revInteger(int num) {
+		int originalNum = num, revInt=0;
+		
+		while(num>0) {
+			revInt = 10*revInt+(num % 10);
```

Comment:
--You can directly write int revInt = 10*revInt+(num % 10);
-- Code logic looks good to me.

Reply:
->If I declare/write revInt like "int revInt = 10*revInt+(num % 10);" as suggested by you, its scope will be limited to while loop only and I will not be able to use it in printing statement i.e. outside while().